src/app/docs/button/button-docs.component.html

```diff
@@ -0,0 +1,196 @@
+<sky-docs-demo-page
+  pageTitle="Buttons"
+>
+  <sky-docs-demo-page-summary>
+    The button classes create buttons to trigger actions from within an interface. For information about styles and implementation, see the <a href="https://developer.blackbaud.com/skyux/design/guidelines/buttons-links">buttons and links guidelines</a>.
```

Comment:
@Blackbaud-AdamFunderburk or @Blackbaud-ToddRoberts, should we delete the second sentence here? We have a link at the bottom of the standard location at the bottom of the Design tab in a *Related guidelines* section. I know we've been trying to minimize the text in the summary, so it seems like we could do without the link to additional information, but let me know what you guys think. Thanks!

Reply:
removing it seems fine to me